docs/user/beyond/fulltext.rst

```diff
@@ -14,6 +14,9 @@ Introduction
 
 Full-text search is for searching a single stored document which is distinguished from regular search based on original texts in database. It tries to match search criteria by examining all of the words in each document. In OpenSearch, full-text queries provided enables you to search text fields analyzed during indexing.
 
+This document describes the original SQL engine. It only applies to queries that cannot be handled by the current SQL engine. Documentation for the current engine can be found  `here <../dql/functions.rst#relevance>`_.
```

Comment:
Officially we use SQL engine v2 for current/new engine. Could you stick with it?

https://github.com/opensearch-project/sql/blob/main/docs/dev/NewSQLEngine.md

Reply:
@MaxKsyunz What about: 
```
This document describes the original SQL engine. It only applies to queries that cannot be handled by `SQL engine V2 <../../dev/NewSQLEngine.md>`. Documentation for full-text search using SQL Engine V2 can be found  `here <../dql/functions.rst#relevance>`_.
```

Reply:
Looks good. Thanks!